examples/lib/defaultHtmlStepUi.js

```diff
@@ -57,7 +57,7 @@ function DefaultHtmlStepUi(_sequencer, options) {
                 <div class="col-md-8 cal collapse in step-column">\
                   <div class="load load-spin" style="display:none;"><i class="fa fa-circle-o-notch fa-spin"></i></div>\
                   <div class="step-image">\
-                    <a class="cal collapse in"><img class="img-thumbnail step-thumbnail"/></a>\
+                    <a class="cal collapse in"><img class="img-thumbnail step-thumbnail" crossorigin="anonymous" /></a>\
```

Comment:
Was this the only thing missing? It was working before but maybe browsers increased security requirements? Wow!

Do you think we can still run a test for this? We could pull in a remote image from a static source and use looks-same to compare it.

Reply:
Ok sir i will try to run this one.
BTW i have one more approach to tackle this problem. We can create a local directory and once we place a url to load the image we can use that url to download the image in that directory and can render image directly from local folder this will remove the overhead of cross-origin.
What's your view?

Reply:
Actually previously this was the error in the console when image failed to load-  

`Uncaught Security Error: Failed to execute ‘toDataURL’ on 
‘HTMLCanvasElement’: tainted canvases may not be `

But after setting `crossorigin=anonymous` this error disappeared.